doc: add Note of options.stdio into child_process (#26604)
nodejs/node: doc/api/child_process.md

   occurred).
6. Positive integer - The integer value is interpreted as a file descriptor
   that is currently open in the parent process. It is shared with the child
   process, similar to how {Stream} objects can be shared. Passing sockets

>    does not supported on Windows.

Reviewer: ```suggestion
is not supported on Windows.
```

6. Positive integer - The integer value is interpreted as a file descriptor
   that is currently open in the parent process. It is shared with the child
   process, similar to how {Stream} objects can be shared.

>    Note: Passing sockets does not support on Windows.

Reviewer 2: How about this?

```suggestion
   Note: Passing file descriptors is not supported on Windows.
```

Author: It seems better than mine! I chenged.

>    Note: Passing file descriptors is not supported on Windows.

Reviewer 3: Please drop the "Note:"

Author: I dropped "Note"!
